### ISSUE TITLE
Comments and opens

### ISSUE THREAD

**Issue by davesnx** [reasonml/reason#2696]
```
// Quick and dirty solution to get updated names
// Can be removed once we switch to controlled inputs
open Webapi.Dom;
```

```
open // Quick and dirty solution to get updated names
     // Can be removed once we switch to controlled inputs
     Webapi.Dom;
```

**Comment by SanderSpies** [comment]
```
// Quick and dirty solution to get updated names
// Can be removed once we switch to controlled inputs
open Webapi.Dom;
```
prints as 
```
// Quick and dirty solution to get updated names
// Can be removed once we switch to controlled inputs
open Webapi.Dom;
```
for me. Or is something else wrong here?

**Comment by davesnx** [comment]
This is resolved under 3.9.0
